utils/src/test/java/org/apache/cloudstack/utils/redfish/RedfishClientTest.java

```diff
@@ -87,15 +87,15 @@ public void validateAddressAndPrepareForUrlTestIpv6() {
     public void buildRequestUrlTestHttpsGetSystemId() {
         RedfishClient redfishclient = new RedfishClient(USERNAME, PASSWORD, true, false, REDFISHT_REQUEST_RETRIES);
         String result = redfishclient.buildRequestUrl(oobAddress, RedfishClient.RedfishCmdType.GetSystemId, systemId);
-        String expected = String.format("https://%s/redfish/v1/Systems/", oobAddress, systemId);
```

Comment:
should not `systemId` be a part of the string ?

Reply:
hm, not intended change. I will investigate why this creeped in.

Reply:
on second thought , I just assumed it worked with the resulting string so the parameter wasn't needed. No idea what redfish actually requires.

Reply:
I guess the pattern is `"https://%s/redfish/v1/Systems/%s"

Reply:
@weizhouapache the tests actually fail this way. Not sure if this is a bug in the test or in the redfish code (`redfishclient.buildRequestUrl(oobAddress, RedfishClient.RedfishCmdType.GetSystemId, systemId);`)